packages/core/src/browser/shell/side-panel-toolbar.ts

```diff
@@ -101,4 +102,13 @@ export class SidePanelToolbar extends BaseWidget {
             this.update();
         }
     }
+
+    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+    showMoreContextMenu(anchor: Anchor): ContextMenuAccess {
```

Comment:
is this for testing purposes only?

Reply:
for now yes

Reply:
it's a side effect, though.
have you considered special marking if the test is the single client?

Reply:
I'm not sure what you mean.